.github/workflows/d3a-interface_ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: d3a-interface-ci
+on:
+  pull_request:
+    branches:
+      - master
+      - staging
+jobs:
+  test-build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      env:
+        TOXENV: travis
```

Comment:
Could be a good time to rename the `travis` tox parameter to something like `ci`

Reply:
nice catch
done